Jellyfin.Server/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -216,8 +192,15 @@ public static string GetResponseContentType(IRequest httpReq)
             {
                 foreach (var acceptsType in acceptContentTypes)
                 {
-                    var contentType = HttpResultFactory.GetRealContentType(acceptsType);
-                    acceptsAnything = acceptsAnything || contentType == "*/*";
+                    // TODO: @bond move to Span when Span.Split lands
+                    // https://github.com/dotnet/corefx/issues/26528
+                    var contentType = acceptsType?.Split(';')[0];
```

Comment:
This also misses `ToLower` and `Trim`.
I think I would prefer to keep the original function.